cram/cram_encode.c

```diff
@@ -2827,8 +2878,8 @@ static int process_one_read(cram_fd *fd, cram_container *c,
 	    cram_stats_add(c->stats[DS_MF], p->mate_flags);
 
 	    cram_stats_del(c->stats[DS_TS], p->tlen);
-	    p->tlen = p->apos - cr->aend;
-	    cram_stats_add(c->stats[DS_TS], p->tlen);
+	    //p->tlen = p->apos - cr->aend;
+	    //cram_stats_add(c->stats[DS_TS], p->tlen);
```

Comment:
Why indeed! The io_lib code even has a "//why?" at the end of those lines. :-)

The answer is simply that I forgot to go back and cross all the 'T's and dot all the 'i's to fully justify my decision of those lines being dead. (Hence commented out rather than removed.)  Sorry, I'll have a further analysis of this.